lib/src/modal/modal-window.component.scss

```diff
@@ -0,0 +1,37 @@
+@import '../sass/colors';
+@import '../sass/functions.scss';
+@import '../sass/mixins.scss';
+
+:host {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    background-color: $white;
+    border-radius: 6px;
+    display: block;
+}
+
+.hc-modal {
+    overflow: auto;
+}
+
+.hc-modal-sm {
+    width: 300px;
+    height: 200px;
```

Comment:
I would remove the height from this all of the hc-modal sizes.

Reply:
This is done. Now the modal body will be at minimum `150px` and will max out 75px less than the view's height. Otherwise, it will expand to fill the content provided.